sprout/tests/tests_views_column_review.py

```diff
@@ -32,7 +33,7 @@ def setUp(self):
             allow_duplicate_value=True,
         )
 
-        file = io.BytesIO(b"TestColumn,Letter\n1,A\n2,B\n3,C")
+        file = io.BytesIO(b"TestColumn,Letter\n1,A\n2,B\n3,C\n4,C")
```

Comment:
```suggestion
        file = io.BytesIO(b"TestColumn,Letter\n1,A\n2,B\n3,C\n4,C\n5,C\n6,C")
```
Might be nice to include that one of the test columns include more than 5 unique values.